### PR TITLE
owncloud-client: 2.3.4 -> 2.4.1

### PR DESCRIPTION
**Pull request by bfortz**
###### Motivation for this change
Version bump

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/owncloud-client/default.nix [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+  version = "2.4.1";
 
   src = fetchurl {
     url = "https://download.owncloud.com/desktop/stable/owncloudclient-${version}.tar.xz";
-    sha256 = "1fpi1mlp2b8sx2993b4mava5c6qw794dmlayih430299z1l9wh49";
+    sha256 = "4462ae581c281123dc62f3604f1aa54c8f4a60cd8157b982e2d76faac0f7aa23";
   };
 
-  patches = [ ../nextcloud-client/find-sql.patch ];
+  patches = [ ./find-sql.patch ];
```
Comment (jtojnar) [inline pkgs/applications/networking/owncloud-client/default.nix]: Would you mind opening a pull request in the [upstream repo](https://github.com/owncloud/client) and adding a link here as a comment?

Comment (bfortz) [reply pkgs/applications/networking/owncloud-client/default.nix]: Well... I made the version bump based on the 2.3.4 version currently in 18.03 stable. This is based on the Enterprise Version from owncloud.com, but there is no way to pull requests. The opensource Standard Edition is the one you mention as upstream repo but it has a different structure and the patch above does not apply. Should we switch to the Standard Edition? (It seems the Enterprise Version has more features, but I don't really care as I'm using only basic functionalities). Maybe @qknight has a take on this?

**Comment by bfortz** [comment]
Any hope to have this merged?

**Comment by xeji** [comment]
@GrahamcOfBorg  build owncloud-client

**Pull request merged**